### ISSUE TITLE
badarith error for "rabbitmqctl await_startup"

### ISSUE THREAD

**Issue by sergey-safarov** [rabbitmq/rabbitmq-server#2158]
I automating rabbitmq initialization in cloud installation.
And found error
```
[root@rmq-a-1 ~]# rabbitmqctl await_startup
warning: the VM is running with native name encoding of latin1 which may cause Elixir to malfunction as it expects utf8. Please ensure your locale is set to UTF-8 (which can be verified by running "locale" in your shell)
Error:
{:badarith, [{:erlang, :div, [:infinity, 100], []}, {:rabbit, :wait_for_boot_to_start, 2, [file: 'src/rabbit.erl', line: 769]}, {:rabbit, :await_startup, 3, [file: 'src/rabbit.erl', line: 760]}, {:rpc, :"-handle_call_call/6-fun-0-", 5, [file: 'rpc.erl', line: 197]}]}
```

To reproduce need execute
```
rabbitmqctl stop_app
```

**Comment by lukebakken** [comment]
Thanks for using RabbitMQ and for reporting this issue. It has already been fixed in https://github.com/rabbitmq/rabbitmq-server/pull/2144

**Comment by sergey-safarov** [comment]
Looks as issue not fixed on 3.8.1
Could you reopen.
```
OS PID: 6188
OS: Linux
Uptime (seconds): 11466
RabbitMQ version: 3.8.1
Node name: rabbit@rmq-a-1.nga911.com
Erlang configuration: Erlang/OTP 21 [erts-10.3.5.6] [source] [64-bit] [smp:2:2] [ds:2:2:10] [async-threads:64] [hipe]
Erlang processes: 250 used, 1048576 limit
Scheduler run queue: 1
Cluster heartbeat timeout (net_ticktime): 60
```
According referenced PR 2144 fix merged into 3.8.x branch and not help.

**Comment by lukebakken** [comment]
Thanks for letting us know what version you're using. Yes, I can reproduce this issue using the steps you provide.

**Comment by michaelklishin** [comment]
Once this change makes it through the pipeline we will produce a `3.8.2` preview for you to use in your automation work.

**Comment by sergey-safarov** [comment]
Where i can download preview rpm build?
Thanks

**Comment by michaelklishin** [comment]
We will post a link here once Concourse produces a new alpha (development) build.

**Comment by michaelklishin** [comment]
[3.8.2-alpha.3](https://bintray.com/beta/#/rabbitmq/all-dev/rabbitmq-server/3.8.2-alpha.3?tab=overview) contains this change. A `3.7.22` alpha is currently making it through the pipeline.

Sorry about the delay, the pipeline was previously blocked by a failing job with an external dependency.

**Comment by michaelklishin** [comment]
I posted an alpha that did not contain this commit, we have to wait a bit more.

**Comment by michaelklishin** [comment]
[3.7.22-alpha.5](https://bintray.com/beta/#/rabbitmq/all-dev/rabbitmq-server/3.7.22-alpha.5?tab=overview) contains this PR. Thanks again.

**Comment by antoinetran** [comment]
Hi, this is just a workaround script for people like me that cannot upgrade Rabbitmq immediately (but in the future yes):
```
            rabbitmq_await() {
              # Workaround of https://github.com/rabbitmq/rabbitmq-server/issues/2158
              # TODO: remove this sleep if rabbitmq >= 3.8.2.
              while ! rabbitmqctl cluster_status &>/dev/null ; do
                sleep 2
              done
              echo "Rabbitmq ready!"
            }
```
